quickrspecpuppet/writer.py

```diff
@@ -51,11 +51,16 @@ def __init__(self, parser):
         self.classes = parser.classes
         self.dependencies = parser.dependencies
         self.fixtures_path = '{0}/{1}'.format(
-            parser._directory, '/.fixtures.yml')
+            parser._directory, '/.fixtures.yml'  # TODO: Why are we using a protected member here?
```

Comment:
Yeah.  Originally, I wasn't using parse._directory externally.  I need to remove the _.